mitmproxy/addons/modifybody.py

```diff
@@ -20,7 +21,7 @@ def load(self, loader):
             """
             Replacement pattern of the form "[/flow-filter]/regex/[@]replacement", where
             the separator can be any character. The @ allows to provide a file path that
-            is used to read the replacement string.
+            is used to read the replacement string.  Streamed bodies cannot be modified.
```

Comment:
I think this addition here may add a bit confusion as it's not clear that this refers to mitmproxy's off-by-default streaming functionality and not say any chunked HTTP encoding. I'd be fine with leaving this out here entirely and just document it in the `stream_large_bodies` option.

Reply:
I have reverted this change